src/app/cloud/_api/fetchTemplates.ts

```diff
@@ -1,12 +1,19 @@
 import { TEMPLATES } from '@root/app/_graphql/templates'
 import type { Template } from '@root/payload-cloud-types'
+import { payloadCloudToken } from './token'
 
 export const fetchTemplates = async (): Promise<Template[]> => {
+  const { cookies } = await import('next/headers')
+  const token = cookies().get(payloadCloudToken)?.value ?? null
+  if (!token) throw new Error('No token provided')
```

Comment:
This is not right, we should not throw an error if there is not a user, we need to instead just not send the token